examples/next-ssr-demo/src/app/server/page.tsx

```diff
@@ -0,0 +1,24 @@
+'server-only'
+
+import './App.css';
+import App from './App';
+import Script from 'next/script';
+import { createPhraseScript } from 'react-intl-phraseapp/functions';
```

Comment:
Let's export `createPhraseScript` in `src/index.ts` so that it's available for one to just do
```
import { createPhraseScript } from 'react-intl-phraseapp';
```
This way, it's consistent with `initializePhraseAppEditor`

I think we can also rename the function to `createPhraseAppEditorScript` or `getPhraseAppEditorScript` for further alignment.

Other than these two small changes, looks good 🔥

Reply:
FYI it's not possible to use it on SSR from the index import, only from `react-intl-phraseapp/functions`

I'm not sure we should include it in `index` for time being since it's more likely to break from there.

<img width="980" alt="Screenshot 2023-09-06 at 15 02 39" src="https://github.com/phrase/react-intl-phraseapp/assets/44475420/f0d6987e-f28b-46a2-bb44-3f1105c66af6">